lib/i18n/tasks/command/commands/translate.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module I18n::Tasks
+  module Command
+    module Commands
+      module Translate
+        include Command::Collection
+        include I18n::Tasks::KeyPatternMatching
+
+        cmd :translate,
+            pos: '[pattern]',
+            desc: t('i18n_tasks.cmd.desc.translate'),
+            args: [:locale, :locale_to_translate_from, arg(:data_format).from(1), :translation_backend]
+
+        def translate(opts = {})
+          forest = i18n.tree(opts[:from])
```

Comment:
@glebm Do you have a suggestion for creating a tree with all keys from `opts[:from]` locale to send into the `translate_forest` method? I have tried a few ways but cannot figure out the correct format.

Reply:
The forest root nodes should be the locales to translate to.

For example, `{"en": {"key": "Value"}}`